block/local/adapter.go

```diff
@@ -93,6 +93,27 @@ func (l *Adapter) Remove(obj block.ObjectPointer) error {
 	return os.Remove(p)
 }
 
+func (l *Adapter) Copy(sourceObj, destinationObj block.ObjectPointer) error {
+	dest := l.getPath(destinationObj.Identifier)
+	destinationFile, err := os.Create(dest)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		_ = destinationFile.Close()
+	}()
+	source := l.getPath(sourceObj.Identifier)
+	sourceFile, err := os.OpenFile(source, os.O_RDONLY, 0755)
```

Comment:
Why not `os.Open`?  (You anyway pass weird permissions, that are then ignored.)

Reply:
👍